icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -112,6 +112,59 @@ class CachingConfig:
     @staticmethod
     def default() -> CachingConfig: ...
 
+class ManifestPreloadCondition:
+    """Configuration for conditions under which manifests will preload on session creation"""
+
+    @staticmethod
+    def or_conditions(
```

Comment:
Can we add doc strings for all these methods?

Reply:
done